ai2thor/video_controller.py

```diff
@@ -386,3 +376,30 @@ def merge_video(self, other_video_path):
                 self.saved_frames.append(rgb)
             success, image = vidcap.read()
             i += 1
+
+    def exportVideo(self, path):
+        """Merges all the saved frames into a .mp4 video and saves it to `path`"""
+        if self.saved_frames:
+            path = path if path[:-4] == '.mp4' else path + '.mp4'
+            if os.path.exists(path):
+                os.remove(path)
+            print((self.saved_frames[0].shape[1], self.saved_frames[0].shape[0]))
+            video = cv2.VideoWriter(
+                path,
+                cv2.VideoWriter_fourcc(*'DIVX'),
+                30,
+                (self.saved_frames[0].shape[1], self.saved_frames[0].shape[0])
+            )
+            for i, frame in enumerate(self.saved_frames):
+                print('|', end='')
+                # assumes that the frames are RGB images. CV2 uses BGR.
+                video.write(cv2.cvtColor(frame, cv2.COLOR_BGR2RGB))
+            cv2.destroyAllWindows()
+            video.release()
+            print('done')
```

Comment:
It looks like `export_video` already exists on the VideoController (See: https://github.com/allenai/ai2thor/blob/main/ai2thor/video_controller.py#L349-L365), and this method appears to be a duplicate of that with some extra print statements? 

Can this be removed?

Reply:
Yes, good point. I was trying to support previous calls to `exportVideo' instead of `export_video'.